src/gluonts/torch/model/predictor.py

```diff
@@ -78,7 +78,7 @@ def predict(
 
         self.prediction_net.eval()
 
-        with torch.no_grad():
+        with torch.inference_mode():
```

Comment:
Looks like this was introduced in torch==1.9.0, at least from the documentation, so this probably requires bumping the requirement? https://github.com/awslabs/gluon-ts/blob/master/requirements/requirements-pytorch.txt